structuredefinitions/UKCore-Observation-InspiredOxygen.xml

```diff
@@ -17,7 +17,7 @@
       <rank value="1" />
     </telecom>
   </contact>
-  <description value="Defines the additional constraints and extensions on the UK Core Observation profile" />
+  <description value="Defines the additional constraints and extensions on the UK Core Observation profile. It basically describes whether a patient requires oxygen or is breathing room air." />
```

Comment:
Change this to something like "Defines the additional constraints and extensions on the UK Core Observation profile, for recording if a patient is breathing room air, or inspired oxygen."

Reply:
Done this Change